styles/_search.scss

```diff
@@ -65,11 +65,22 @@ h2 .faint {
   margin-left: 20px;
 }
 
+// #search-hint{
```

Comment:
This is my own proposed interaction for adding a #search-hint much like the one on github, it's messing with CSS currently so commented but I left in for future use.
![screen shot 2018-06-05 at 9 46 00 am](https://user-images.githubusercontent.com/29494303/40990359-6a32fb74-68a5-11e8-8aec-dc8182eda119.png)